.reaction/jsdoc/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+## runs jsdoc
+## master branch docs will be published via aws s3 sync when all test pass
+set -e
+
+## install awscli
+apt-get -y -qq install awscli
+## install jsdoc
+npm install -g jsdoc
+# build new jsdocs
+echo "Running jsdocs on the reaction codebase"
+jsdoc . --verbose --configure .reaction/jsdoc/jsdoc.json
```

Comment:
do we want to use `node_modules/.bin/jsdoc` instead of `jsdoc` here?

Reply:
Nope.  Should be fine as it is since `jsdoc` is being installed globally on line 9.